lessons/module-3/react-2-the-how.md

```diff
@@ -478,20 +462,20 @@ const Ideas = (props) => {
 
 What are those curly brackets doing? In JSX, whenever we're writing something that is JavaScript (aka "not HTML"), we have to wrap it in curly brackets. In this case, "name" acts like a variable. It's not a string that reads "name" - it's a placeholder that represents the value of the property (in this case, "Travis")! Because it's a variable, we have to surround it in curly brackets to tell the JSX to treat the contents like JavaScript.
 
-In your browser, you should see "Hello, Travis!" In `App.js`, add another Ideas component to the `render` method, but pass in a different name. What do you see in the browser? Try creating new props to use!
+In your browser, you should see "Hello, Travis!" In `App.js`, add another Ideas component to the `App.js` `return()` , but pass in a different name. What do you see in the browser? Try creating new props to use!
 
 Okay, so just WHAT exactly is going on here?
 
-Similar to how **state** is an object that contains key-value pairs, **props** is the name of an object that contains key-value pairs. From our above example, the key is "name", and the value is "Travis". So, in our Ideas component, we can access the value by writing `props.name` (which gives us a string of "Travis"). This is the same dot notation we learned in Mods 1 and 2 to access data stored in objects.
+ **props** is the name of an object that contains key-value pairs. From our above example, the key is "name", and the value is "Travis". So, in our Ideas component, we can access the value by writing `props.name` (which gives us a string of "Travis"). This is the same dot notation we learned in Mods 1 and 2 to access data stored in objects.
 
-If, in the `render` method of our App component, we called the property "potato" instead of "name", we would have to access it by (inside the Ideas component) writing `props.potato`.
+If, in the `return` of our App component, we called the property "potato" instead of "name", we would have to access it by (inside the Ideas component) writing `props.potato`.
 
 We can even destructure the props object, because it's just a regular object!
 
 ```js
 // Ideas.js
 
-const Ideas = (props) => {
+function Ideas(props){
```

Comment:
Great question, @Kalikoze ! The React docs is using the function keyword, but I'm not sure if there are any preferences here. At first, I was using the function keyword, but I decided to change them to expressions. I thought that maybe we could use function keywords while teaching lessons so that students can see both.